bims/utils/user_permissions.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf-8
+from braces.views import LoginRequiredMixin
+from django.http import Http404
+
+
+class ValidatorRequiredMixin(LoginRequiredMixin):
+
+    def dispatch(self, request, *args, **kwargs):
+        validator = \
+            'validator' in request.user.groups.all().values_list(
+                'name', flat=True)
+
+        if not request.user.is_authenticated():
```

Comment:
I think this is already handled in LoginRequiredMixin

Reply:
I extended the class LoginRequiredMixin so that it will check first whether a user is login or not, then once login it will check whether this user is in the `validator` group or not